handlers/utils.go

```diff
@@ -17,7 +18,7 @@ type promClientSupplier func() (*prometheus.Client, error)
 var defaultPromClientSupplier = prometheus.NewClient
 
 func checkNamespaceAccess(nsServ business.NamespaceService, namespace string) (*models.Namespace, error) {
-	if nsInfo, err := nsServ.GetNamespace(namespace); err != nil {
+	if nsInfo, err := nsServ.GetNamespace(context.TODO(), namespace); err != nil {
```

Comment:
I understand that we need to propagate the context from the request to pass the trace context.
I've just wondered if some of this TODO is missing the context.
I'd pass the context in all the cases from the handlers ?

Reply:
Yeah only some of the handlers were updated to pass the context through. I thought it would keep the PR smaller and I wanted to gauge interest on the changes a bit before updating all the handlers at once. I'm happy to go back and update them though as part of this PR.